natchez-extras-doobie/src/main/scala/com/ovoenergy/natchez/extras/doobie/TracedStatement.scala

```diff
@@ -14,7 +14,8 @@ import scala.annotation.nowarn
 @nowarn
 private[doobie] case class TracedStatement(
   p: PreparedStatement,
-  queryString: String
+  queryString: String,
+  c: Any*
```

Comment:
What does `c` stand for? Could you please use a more descriptive name?

I see that we didn't set a good example with `p` for `preparedStatement` on line 16, but we should do better in the future.